Common subdirectories: slocate-2.5.orig/doc and slocate-2.5/doc
diff -u slocate-2.5.orig/main.c slocate-2.5/main.c
--- slocate-2.5.orig/main.c	Sat Dec 30 08:35:30 2000
+++ slocate-2.5/main.c	Sat Feb 17 03:40:16 2001
@@ -417,6 +417,8 @@
 
 		*regexp = 0;
 		strcat(regexp,ptr);
+		if (ADD_SLOCATEDB)
+		    parse_decode_path(SLOCATEDB);
 		while ((database = SLOCATE_PATH[i++])) res |= decode_db(database, ptr);
 		exit(!res);
 	}
@@ -1411,6 +1413,8 @@
 			    report_error(FATAL,QUIET,"%s: main: 'regexp': malloc: %s\n",progname,strerror(errno));
 			*regexp = 0;
 			strcat(regexp,optarg);
+			if (ADD_SLOCATEDB)
+			    parse_decode_path(SLOCATEDB);
 			while ((database = SLOCATE_PATH[i++])) res |= decode_db(database, optarg);
 			return(res);
 			break;
